charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,14 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+      - name: ghostunnel-server
+        command: ["ghostunnel"]
+        args: ["server", "--listen", "localhost:8443", "--target", "localhost:9083", "--key", "/opt/hive/tls/tls.key", "--cert", "/opt/hive/tls/tls.crt", "--cacert", "/opt/hive/tls/ca.crt", "--allow-cn", "Any"]
+        image: quay.io/openshift/origin-ghostunnel
+        volumeMounts:
+        - name: hive-metastore-secret
+          mountPath: /opt/hive/tls
+          readOnly: true
```

Comment:
secret volume mounts are `readOnly: true` by default. See: https://github.com/kubernetes/kubernetes/issues/62099#issuecomment-378809922

Reply:
When I looked at it by way of kubectl get secrets it was read write